src/main/java/org/broadinstitute/consent/http/models/darsummary/DARModalDetailsDTO.java

```diff
@@ -82,97 +73,104 @@ public boolean isSensitivePopulation() {
         return sensitivePopulation;
     }
 
-    public List<SummaryItem> generatePurposeStatementsSummary(Document darDocument) {
+    public List<SummaryItem> generatePurposeStatementsSummary(DataAccessRequest dar) {
         List<SummaryItem> psList = new ArrayList<>();
-        if(darDocument.getBoolean(DarConstants.FOR_PROFIT)){
-            psList.add(new SummaryItem(SummaryConstants.PS_FOR_PROFIT, false));
-        }
-        if(darDocument.getBoolean(DarConstants.ONE_GENDER)) {
-            if (darDocument.getString(DarConstants.GENDER).equalsIgnoreCase("F")) {
-                psList.add(new SummaryItem("Gender: ", SummaryConstants.PS_GENDER_FEMALE, false));
-            } else {
-                psList.add(new SummaryItem("Gender: ", SummaryConstants.PS_GENDER_MALE, false));
+        if (Objects.nonNull(dar) && Objects.nonNull(dar.getData())) {
+
+            if (dar.getData().getForProfit()) {
+                psList.add(new SummaryItem(SummaryConstants.PS_FOR_PROFIT, false));
+            }
+            if (dar.getData().getOneGender()) {
```

Comment:
Here and below. I believe that all properties on dar.getData() can be null which will throw errors in all of these if blocks. Consider using something like `org.apache.commons.lang.BooleanUtils.isTrue` ([link](https://www.oschina.net/uploads/doc/commons-lang-2.4/org/apache/commons/lang/BooleanUtils.html#isTrue(java.lang.Boolean)))

Reply:
Alternatively, inside each of the getters for all of these booleans, you can use `BooleanUtils.isTrue` on the class field which would allow you to make all of these calls without the additional boilerplate.

Reply:
A third option is to create a private utility method that does the same thing, i.e., 
```
private boolean isTrue(Boolean obj) {
    return BooleanUtils.isTrue(obj);
}
```
which would not introduce any other side effects if you added code to the getters in DarData.